src/sidebar/util/test/build-thread-test.js

```diff
@@ -449,14 +456,13 @@ describe('sidebar/util/build-thread', function () {
       ];
 
       const thread = createThread(fixture, {
-        sortCompareFn: function (a, b) {
-          return a.updated > b.updated;
-        },
+        sortCompareFn: (a, b) => b.annotation.updated - a.annotation.updated,
```

Comment:
~~The updated field is an ISO date string so subtracting will produce `NaN`.~~

Edit: I see that `updated` is a number in this context, so this works, although it is misleading because `updated` is not a number in the real app.